velox/exec/DistinctAggregations.cpp

```diff
@@ -113,7 +126,11 @@ class TypedDistinctAggregations : public DistinctAggregations {
         }
 
         rows.resize(data->size());
-        aggregate.function->addSingleGroupRawInput(group, rows, {data}, false);
+        std::vector<VectorPtr> args = {data};
```

Comment:
Let's initialize the vector to {data} only when there is single input.

Consider adding helper method makeInputForAggregation to match makeInputForAccumulator.